spec/function.dd

```diff
@@ -592,6 +592,9 @@ $(H2 $(LNAME2 optional-parenthesis, Optional Parentheses))
 
 $(H2 $(LNAME2 property-functions, Property Functions))
 
+    $(P WARNING: The future of property functions is uncertain.  Using property functions is
+    discouraged until the feature can be properly designed and implemented.)
```

Comment:
I would be a bit less dramatic:

"The definition and usefulness of `@property` function is being reviewed, and their implementation is currently incomplete w.r.t the current specifications. Using property functions is not recommended until the feature [...]"

Reply:
Done.